latrine_django/latrineapp/models.py

```diff
@@ -1,3 +1,42 @@
 from django.db import models
 
-# Create your models here.
+
+class Place(models.Model):
```

Comment:
why doesn't 'resources'  have to be in the Place model?

Reply:
Each place can have many resources. For example, the Burnside shelter can have both showers and restrooms. Each facility can be displayed on the map separately, with its own comments and flags. That's why we need a separate Resource model, connected to the Place model via a foreign key.